plugins/vpc-bridge/config/k8s/connector.go

```diff
@@ -0,0 +1,73 @@
+// Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"). You may
+// not use this file except in compliance with the License. A copy of the
+// License is located at
+//
+//     http://aws.amazon.com/apache2.0/
+//
+// or in the "license" file accompanying this file. This file is distributed
+// on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+// express or implied. See the License for the specific language governing
+// permissions and limitations under the License.
+
+//go:build enablek8sconnector && windows
+
+package k8s
+
+import (
+	"context"
+	"fmt"
+
+	log "github.com/cihub/seelog"
+)
+
+// k8sConnector Represents k8s connector to get Pod IP using connector binary and pipe.
+type k8sConnector struct {
```

Comment:
Can we please remove this like discussed.

This package should have one method:

`GetPodIP(ctx, namespace, podName) (string, error)`

Internally this can just do what `GetPodIP` is doing which is:

 ```
 result := make chan(resulttype)
go readResultFromPipe(ctx, namespace, podName, result)
err, out := executek8sConnector(ctx, namespace, podName)
.. handle errors
select {
case <-ctx.Done():
case r:= <-result:
    // handle result
}

Reply:
done